### ISSUE TITLE
Options to clear shell, and clear history

### ISSUE THREAD

**Issue by casonadams** [austinjones/tab-rs#270]
```
15:54:51.274 ~/Projects/tab-rs [0]
⦔ clear
terminals database is inaccessible
```

Because sometimes you want to clear the shell and or the history.

**Comment by austinjones** [comment]
I think this is a bug related to the $TERM env var.  I'll take a look!

**Comment by austinjones** [comment]
Hey @casonadams.  I've merged some code to `main` that should fix this (#280).  I couldn't reproduce it though - is any chance you could run `scripts/run.sh test270` and see if you can clear your terminal?  (no need to shut down the daemon - and any unique tab name would work)

**Comment by casonadams** [comment]
@austinjones seems to still be happening
```
12:44:43.026 ~/Projects/tab-rs [test270]
⦔ clear
terminals database is inaccessible
12:44:45.199 ~/Projects/tab-rs [test270]
⦔ echo $TERM
tmux-256color
12:45:25.575 ~/Projects/tab-rs [test270]
⦔
```

**Comment by austinjones** [comment]
Hey @casonadams ,
I think the issue here is the `tmux-256color` value - which seems to have problems on OSX w/ terminfo.  See this tmux bug report: https://github.com/tmux/tmux/issues/1257

If you can figure out where it's coming from, xterm-256color should work.  I think all I can do in tab is pass the $TERM var along.

**Comment by austinjones** [comment]
Hey @casonadams.  I just released [v0.5.4] with the $TERM fix.  Did you find anything else while looking at this that might be useful?  One thing I was wondering is if $TERMINFO was different inside/outside the session.

**Comment by austinjones** [comment]
I'm going to close this.  I can't think of any other changes that might fix it.  Let me know if you have more info, though!
